src/hotspot/share/classfile/moduleEntry.cpp

```diff
@@ -33,7 +33,6 @@
 #include "memory/archiveUtils.hpp"
```

Comment:
Do these files need archiveUtils.hpp or does archiveBuilder.hpp need it?

Reply:
Both packageEntry.cpp and moduleEntry.cpp use ArchivePtrMarker which is defined in archiveUtils.hpp.